src/main/java/org/jabref/gui/shared/SharedDatabaseUIManager.java

```diff
@@ -81,9 +84,40 @@ public void listen(UpdateRefusedEvent updateRefusedEvent) {
 
         jabRefFrame.getDialogService().notify(Localization.lang("Update refused."));
 
-        new MergeSharedEntryDialog(jabRefFrame, dbmsSynchronizer, updateRefusedEvent.getLocalBibEntry(),
-                                   updateRefusedEvent.getSharedBibEntry(),
-                                   updateRefusedEvent.getBibDatabaseContext().getMode()).showMergeDialog();
+        BibEntry localBibEntry = updateRefusedEvent.getLocalBibEntry();
+        BibEntry sharedBibEntry = updateRefusedEvent.getSharedBibEntry();
+
+        StringBuilder message = new StringBuilder();
+        message.append(Localization.lang("Update could not be performed due to existing change conflicts."));
+        message.append("\r\n");
+        message.append(Localization.lang("You are not working on the newest version of BibEntry."));
+        message.append("\r\n");
+        message.append(Localization.lang("Shared version: %0", String.valueOf(sharedBibEntry.getSharedBibEntryData().getVersion())));
+        message.append("\r\n");
+        message.append(Localization.lang("Local version: %0", String.valueOf(localBibEntry.getSharedBibEntryData().getVersion())));
+        message.append("\r\n");
+        message.append(Localization.lang("Please merge the shared entry with yours and press \"Merge entries\" to resolve this problem."));
```

Comment:
Shouldn't this be something like "Press \"Merge entries\" to resolve the merge conflict." ?

Reply:
Yeah. Fixed the wording